main.py

```diff
@@ -0,0 +1,87 @@
+import random
+import os
+from art import logo
+
+# function calculateScore takes in a list parameter and returns sum of list
+def calculateScore(list):
+    count = sum(list)
+
+    if count == 21:
+        return 21
+    elif count > 21:
+        for num in list:
+            if num == 11:
+                list.remove(num)
+                list.append(1)
+                count = sum(list)
+    
+    return count
+
+# function compareScrore takes in user and dealer's scores, compares them, and returns a string
+def compareScore(userScore, dealerScore):
```

Comment:
PEP-8 (look it up) conventions for Python variables and functions is all `snake_case`.  Please fix.

Reply:
You also need to type-cast all input variables like so:

def filter_object_fields(object: dict, included: List[str] = [], excluded: List[str] = []) -> dict:

It makes a world of difference when you're trying to debug.

Reply:
I included the snake_case, and will continue with that format from now on.
As for the type-casting, I couldn't find any relevant resources or examples where it is used in functions like the example above. Hopefully you can explain this to me in more detail at a later time

Reply:
@mercurrie -- type-casting means that you identify the type (e.g., str, list) of the inputs and outputs like so:

def filter_object_fields(object, included, excluded):

VS:

def filter_object_fields(object: dict, included: List[str] = [], excluded: List[str] = []) -> dict:

What it does is allow the IDE to easily identify if there's any errors that might appear caused by incorrect type's before you run the code.  Use the function above as an example and see if you can specify what every input and output of every function should be.